### PR TITLE
Update

### PR DESCRIPTION
**Pull request by bbhtt**
Please see the individual commits for the actual change and ignore the indentation change. It created a lot of noise.

Ping @ondratu please review, thanks!

### CODE REVIEW

**Comment by bbhtt** [comment]
I'll fix further issues later.

File: cz.zeropage.Formiko.json [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
     "rename-desktop-file": "formiko.desktop",
     "rename-appdata-file": "formiko.metainfo.xml",
     "finish-args" : [
         "--share=ipc",
-        "--socket=x11",
+        "--device=dri",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--share=network",
         "--filesystem=home",
```
Comment (bbhtt) [inline cz.zeropage.Formiko.json:16]: Is this really needed or can this be reduced to a subset of `$HOME` like XDG directories? I have opened an issue about using portals https://github.com/ondratu/formiko/issues/39

File: cz.zeropage.Formiko.json [hunk 2]
```diff
@@ -31,253 +35,43 @@
...
                 }
             ]
         },
+        "python3-modules.json",
         {
             "name": "formiko",
             "buildsystem": "simple",
             "build-commands": [
-                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} formiko"
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --root=/ --optimize=1"
```
Comment (ondratu) [inline cz.zeropage.Formiko.json:65]: Why is installing from source code (git) via setup.py better, than installing via pip from python wheel?

File: cz.zeropage.Formiko.json (hunk 1 above)
Comment (ondratu) [reply cz.zeropage.Formiko.json:16]: I replied to your issue. At this moment it is necessary for preview widget.

File: cz.zeropage.Formiko.json (hunk 2 above)
Comment (bbhtt) [reply cz.zeropage.Formiko.json:65]: I did it mostly to differentiate between the python dependencies and the actual app being packaged; and it's also a personal preference. Also it'd be easier to apply small patches quickly if the build somehow breaks (broken appdata/icon issue etc.) during runtime/dependency bumps than waiting for a release.

File: cz.zeropage.Formiko.json [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
-        "--socket=x11",
+        "--device=dri",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--share=network",
         "--filesystem=home",
         "--filesystem=/media",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=xdg-run/gvfsd",
+        "--talk-name=org.gtk.vfs",
```
Comment (hfiguiere) [inline cz.zeropage.Formiko.json:20]: this line should never have been added.

Comment (bbhtt) [reply cz.zeropage.Formiko.json:20]: https://github.com/flathub/cz.zeropage.Formiko/pull/4

**Pull request merged**